app/models/authorization_definition.rb

```diff
@@ -45,6 +46,7 @@ def self.build(uid, hash)
         stage: Stage.new(hash[:stage]),
         scopes: (hash[:scopes] || []).map { |scope_data| AuthorizationDefinition::Scope.new(scope_data) },
         blocks: hash[:blocks] || [],
+        features: (hash[:features] || {}).transform_keys(&:to_sym),
```

Comment:
```suggestion
        features: Hash(hash[:features]).transform_keys(&:to_sym),
```

d'ailleurs ça pourrait scout les tableaux aussi cf L47 avec Array()

et totalement nitpick mais pourquoi pas `.with_indifferent_access` ?
C'est surement moins "performant" mais ça me semblerait plus "rails" :sweat_smile:

Reply:
C'est tellement du détail++ que je fais pas gaffe. tbh ça m'est égal.